auto_landing/include/auto_landing/eight_shape_traj.hpp

```diff
@@ -0,0 +1,32 @@
+#include<ros/ros.h>
+
+#include<geometry_msgs/PoseStamped.h>
+#include<geometry_msgs/Twist.h>
+
+#include<nav_msgs/Odometry.h>
+
+namespace ariitk::auto_landing {
+
+class Eight_Traj {
```

Comment:
Class names should be CamelCased : refer https://github.com/AerialRobotics-IITK/ariitk_common/blob/master/GUIDE.md